src/Service/AlgoliaQuerier.php

```diff
@@ -76,6 +90,16 @@ function array_key_first(array $arr)
             }
         }
 
+        //Custom sorting results so Guide pages are always first.
+        //This will unset all results that aren't guide pages
+        //Then instantly re add to array which places them at the bottom.
+        foreach($records as $key => $wanted) {
+            if($wanted->ClassName != 'IntranetGuidePage') {
```

Comment:
@phillygogo Definitely should not be done in here :) Sorting can be set in Algolia directly.